app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,12 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+  end
+
+  def add_room
```

Comment:
The content of this method is correct, but I wouldn't use an action called `add_room` since it's not part of the 7 RESTful routes (`index, show, new, create, edit, update, delete`); rather what we're doing in this method is _creating_ an instance of a `Booking` model that you made. So, a better name for this action would be `create`. 

Also, since we're creating a Booking and not a Guest, it really belongs in a `BookingController` file. But once that's created, we could just lift the code in this method right over there, and it would work great there too.